script.js

```diff
@@ -84,19 +63,12 @@ document.addEventListener("DOMContentLoaded", () => {
 
     for (let item of fetchedData) {
       const tr = createEl({ el: "tr" });
-      const td = createEl({ el: "td", styles: `padding: 10px;` });
-      const input = createCheckbox({
-        el: "input",
-        type: "checkbox",
-        attribute: "id",
-        attributeVal: item.id,
-      });
-      td.append(input);
-      tr.append(td);
-      thNames.slice(1).forEach((thName) => {
+      thNames.forEach((thName) => {
         const td = createEl({
           el: "td",
-          text: createUniqTd(thName, item.body, unique, item),
+          text: createUniqTd(
+            new TdDto({ name: thName, body: item.body, cb: unique, item })
```

Comment:
Replace with a plain object. It may a classical realization of dto, but we don't need it in the JS (creating class). Use plain objects